machinelearning/tensorflow.md

```diff
@@ -0,0 +1,349 @@
+# ML inference on Graviton CPUs with TensorFlow
+
+**Introduction**
+
+TensorFlow is an open-source software library for machine learning and artificial intelligence. It can be used across training and inference of deep neural networks. This document covers how to enable Tensorflow based machine learning inference on Graviton CPUs, what runtime configurations are important and how to debug any performance issues. The document also covers the instructions source builds to enable downstream features.
+
+# How to enable Tensorflow on Graviton CPUs
```

Comment:
enable or use?

Reply:
I think 'use' is more appropriate, will fix it.